# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../vbetool/vbetool-1.1-linking.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Decription: Do not link explicitely against libpci.a

--- vbetool-1.1/Makefile.am.orig	2011-05-12 10:45:05.264002074 +0200
+++ vbetool-1.1/Makefile.am	2011-05-12 10:46:00.696002074 +0200
@@ -2,7 +2,7 @@
 
 sbin_PROGRAMS = vbetool
 
-vbetool_LDADD = $(libdir)/libpci.a
+vbetool_LDADD = -lpci -lz -lx86
 
 man_MANS = vbetool.1
 vbetool_SOURCES = vbetool.c $(x86)
@@ -14,4 +14,3 @@
 	$(RM) Makefile.in aclocal.m4 config.h.in stamp-h.in configure
 
 AM_CFLAGS = -g -Wall -pedantic -std=gnu99
-AM_LDFLAGS = -lz -lx86
